doubleml/utils/_tune_optuna.py

```diff
@@ -286,18 +286,25 @@ def _get_optuna_settings(optuna_settings, params_name=None):
 
     # Find matching learner-specific settings, handles the case to match ml_g to ml_g0, ml_g1, etc.
     learner_specific_settings = {}
-    if any(params_name in key for key in learner_or_params_keys):
```

Comment:
wasnt only the `params_name` wrong? Shouldnt it be key in params_name? Shouldnt 

```
        for k in learner_or_params_keys:
            if k in params_name and params_name != k:
                learner_specific_settings = optuna_settings[k]
```
be sufficient?